### PR TITLE
Team 1 rspec organization

### PR DESCRIPTION
**Pull request by EricRichardson**
We made the recommended changes to our last pull request.

### CODE REVIEW

File: app/models/organization.rb [hunk 1]
```diff
@@ -46,4 +47,8 @@ def self.search(params)
       all
     end
   end
+
+  def reasonable_tech_team_size
+    errors.add(:tech_team_size, "Team size must be less than organization size") unless tech_team_size < employee_count
```
Comment (whilestevego) [inline app/models/organization.rb]: This line is too long. Make this conditional multi-line.

``` ruby
if not tech_team_size < employee_count
  #code
end
```

**Comment by whilestevego** [comment]
I only saw one thing to fix. Fix then ping me. Should be good to merge afterwards! 😄

**Comment by EricRichardson** [comment]
Fixed

**Comment by whilestevego** [comment]
@EricRichardson looks good! But, a merge that happened before you caused conflicts. Resolve then it should be good to go

**Pull request merged**